google/web-stories-wp
# Page Templates: Always show images when applying default template

## Context


In the default templates layout, we no longer want to show the placeholder grid image. Instead we want to show the default images that are associated with the template. 

If a user has not seen the help center tip about adding background images, we want to show that tip when they add a default template as a page. 

## Summary



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:


1. Add a default template as a page in the editor.
2. You should now see any and all images on the canvas that were present in the thumbnail view. (No placeholder grid image)


## Reviews

### Does this PR have a security-related impact?
no


### Does this PR change what data or activity we track or use?
no



### Does this PR have a legal-related impact?
no


## Checklist




---



Fixes #9536
